packages/react/src/hooks/useResponsiveTypography.ts

```diff
@@ -0,0 +1,83 @@
+import { BreakpointsType, TypographyStyle, useTheme } from '@douro-ui/react';
```

Comment:
Inside react package you don't need to point to the package name, just use:

'../'